robottelo/rhsso_utils.py

```diff
@@ -50,16 +45,16 @@ def get_rhsso_client_id():
         '--password {}'.format(
             KEY_CLOAK_CLI,
             settings.rhsso.host_url.replace('https://', 'http://'),
-            realm,
-            rhsso_user,
-            rhsso_password,
+            settings.rhsso.realm,
+            settings.rhsso.rhsso_user,
+            settings.rhsso.rhsso_password,
         ),
-        hostname=rhsso_host,
+        hostname=str(settings.rhsso.host_name),
```

Comment:
These `str()` wrappers can be removed, its an artifact of the SettingsNodeWrapper behavior.